src/designer/elsa-workflows-studio/src/components/controls/elsa-input-tags/elsa-input-tags-dropdown.css

```diff
@@ -0,0 +1,17 @@
+:host {
+    display: block;
+}
+
+.tag-input:focus {
+    outline: none !important;
+    box-shadow: none !important;
+}
+
+.pr-8 {
+    padding-right: 2rem !important;
+}
+
+.bg-green-350 {
+    --tw-bg-opacity: 1;
+    background-color: rgba(52,191,113,var(--tw-bg-opacity));
+}
```

Comment:
Instead of copying Tailwind classes, you might do this instead:

1. On your local machine, run `npm run build:tailwind:dev` - this will generate a big Tailwind CSS file useful for local development.
2. Now you can just use any TW class as you need.

Unless there's something I'm missing?

Reply:
Yes, good suggestion, just committed this change.
Trying to work on other suggestions.